app/code/Magento/StoreGraphQl/Model/Resolver/Store/StoreConfigDataProvider.php

```diff
@@ -55,11 +55,10 @@ public function __construct(
      */
     public function getStoreConfigData(StoreInterface $store): array
     {
-        $storeConfigData = array_merge(
-            $this->getBaseConfigData($store),
-            $this->getExtendedConfigData((int)$store->getId())
+        return array_merge(
```

Comment:
I don't think we should swap these, it will change behavior in a way we may not want. If store_name is getting overwritten, you should remove this in di.xml
``` <type name="Magento\StoreGraphQl\Model\Resolver\Store\StoreConfigDataProvider">
        <arguments>
            <argument name="extendedConfigData" xsi:type="array">
                <item name="store_name" xsi:type="string">store/information/name</item>
            </argument>
        </arguments>
    </type>```